src/registrar/templates/includes/footer.html

```diff
@@ -26,6 +26,10 @@
           >
             <address class="usa-footer__address">
```

Comment:
The mobile grid classes could use some (probably easy) tweaking for a better mobile layout.

Before:

![Screen Shot 2023-12-05 at 8 12 31 PM](https://github.com/cisagov/manage.get.gov/assets/107004823/dfedad15-a467-48f9-bbb3-20004d169471)


After (not really tested, just browser fiddling):
![Screen Shot 2023-12-05 at 8 12 23 PM](https://github.com/cisagov/manage.get.gov/assets/107004823/55745e57-ab2c-46a8-85cd-3672d389b776)

Reply:
This seems to only happen when you really squish the screen on desktop -- it seems on most common devices it's fine (ie iPhone + Android)

With an iPhone SE:
<img width="446" alt="Screenshot 2023-12-05 at 5 31 47 PM" src="https://github.com/cisagov/manage.get.gov/assets/13954664/4a2851b1-382a-47cb-a9f6-43abd90090d5">

On Samsung Galaxy S20 Ultra:
<img width="574" alt="Screenshot 2023-12-05 at 5 32 19 PM" src="https://github.com/cisagov/manage.get.gov/assets/13954664/e050f44b-78df-477e-9759-bf7b44b98073">

Although I do see that for the Surface Duo it does present that same stacking set up:
<img width="592" alt="Screenshot 2023-12-05 at 5 33 28 PM" src="https://github.com/cisagov/manage.get.gov/assets/13954664/e77ed034-c5bf-41e0-9a7f-553f8a8b650b">

I'm not pressed to add it in this PR, but can be convinced 😅

Reply:
Agree and defer to the designers @Katherine-Osos @gabydisarli. This can be a low priority quick fix ticket for me to have fun with in the future.

Reply:
Unless it's easy to fix now, I'm fine with this being a new ticket to adjust later.

Reply:
Let me spend 10-15 on this, and if I can't figure it out I'll leave it to be a low priority quick fix later if that's alright!

Reply:
Timeboxed and couldn't resolve it -- I've opened a [quick fix ticket](https://github.com/cisagov/manage.get.gov/issues/1443) for it

Reply:
Quick fix/small. It's easy but it's also fiddly to get it right in all screen sizes. Happy to tackle in the future.